index.js

```diff
@@ -116,6 +115,14 @@ AFRAME.registerComponent('controller-cursor', {
   },
 
   /**
+   * Remove mesh.
+   */
+  remove: function () {
+    var cursorEl = this.el;
+    cursorEl.removeObject3D('cursormesh');
```

Comment:
Good question, it looks like `pause` is called by a-frame itself prior to `remove`. I tested this only with console log comments (it showed both pause and remove functions running when the component is removed), however I have not explicitly tested to see if the event listeners are still running upon removing the component.